internal/tiltfile/tiltfile_state.go

```diff
@@ -163,7 +166,7 @@ func (s *tiltfileState) loadManifests(absFilename string, userConfigState model.
 		s.k8sContextExt,
 		dockerprune.NewExtension(),
 		analytics.NewExtension(),
-		version.NewExtension(),
+		version.NewExtension(s.tiltVersion.Version),
```

Comment:
can we have `wire` dependency-inject this? that way we don't have to maintain the handwritten dependency-injection of Version. see how k8sContextExt works.

ideally, we should probably just have a wire provider that creates all the extensions.